src/Controller/FormController.php

```diff
@@ -266,4 +266,23 @@ private function convertFormDataForRequest(array $formData, string $prefix = '')
 
         return $items;
     }
+
+    /**
+     * Remove indices in associative arrays if they are all integers.
+     * This is "necessary" when we want to add a new item to a collection without overwriting an existing element.
```

Comment:
```suggestion
     * This is "necessary" when we want to add a new item to a collection without overwriting an existing element.
     * Especially because of JavaScript: `JSON.parse('{"1": "1", "0":"0"}');` = `{0: '0', 1: '1'}`.
```